package-lock.json

```diff
@@ -3578,6 +3589,21 @@
         "uuid": "^7.0.3"
       },
       "dependencies": {
+        "strong-globalize": {
```

Comment:
this will cause issues again. check where we have strong-globalize otherwise. It should be one version all thru.

Reply:
`strong-globalize` is used by multiple packages internally. New package-lock only has 2 versions (5.1.0 and 6.0.5) installed.